packages/react-flex/README.md

```diff
@@ -0,0 +1,15 @@
+# @fluentui/react-flex
+
+**React Flex components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
```

Comment:
Can you update this description? Probably something like "Flexbox abstraction for..."